runtime/kusama/src/lib.rs

```diff
@@ -115,7 +115,7 @@ pub const VERSION: RuntimeVersion = RuntimeVersion {
 	impl_name: create_runtime_str!("parity-kusama"),
 	authoring_version: 2,
 	spec_version: 9090,
-	impl_version: 0,
+	impl_version: 1,
```

Comment:
this is resolved in master, no need for bump

Reply:
thx